corehq/apps/repeaters/views.py

```diff
@@ -26,13 +31,69 @@ def dispatch(self, request, *args, **kwargs):
     def page_url(self):
         return reverse(self.urlname, args=[self.domain])
 
-    def make_repeater(self):
-        repeater = super(AddCaseRepeaterView, self).make_repeater()
+    def set_repeater_attr(self, repeater, cleaned_data):
+        repeater = super(AddCaseRepeaterView, self).set_repeater_attr(repeater, cleaned_data)
         repeater.white_listed_case_types = self.add_repeater_form.cleaned_data['white_listed_case_types']
         repeater.black_listed_users = self.add_repeater_form.cleaned_data['black_listed_users']
         return repeater
 
 
+class EditRepeaterView(AddRepeaterView):
```

Comment:
All methods in `EditRepeaterView` are over writing methods from `AddRepeaterView`. Not sure if extracting all other methods in a mixin would make this better or may be a `BaseRepeaterView` class would be better?